src/Aks/Aks.csproj

```diff
@@ -1,73 +1,59 @@
-    <Prefer32Bit>false</Prefer32Bit>
+    <AssemblyOriginatorKeyFile>..\MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
+    <DefineConstants>TRACE;RELEASE;NETSTANDARD;SIGN</DefineConstants>
   </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Update="Properties\Resources.Designer.cs">
+      <DesignTime>true</DesignTime>
+      <AutoGen>true</AutoGen>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Update="Properties\Resources.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+  </ItemGroup>
+
   <ItemGroup>
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Version2017_08_31\ContainerServiceClient.cs" />
-    <Compile Include="Version2017_08_31\IContainerServiceClient.cs" />
-    <Compile Include="Version2017_08_31\IManagedClustersOperations.cs" />
-    <Compile Include="Version2017_08_31\ManagedClustersOperations.cs" />
-    <Compile Include="Version2017_08_31\ManagedClustersOperationsExtensions.cs" />
-    <Compile Include="Version2017_08_31\Models\ContainerServiceAgentPoolProfile.cs" />
-    <Compile Include="Version2017_08_31\Models\ContainerServiceDiagnosticsProfile.cs" />
-    <Compile Include="Version2017_08_31\Models\ContainerServiceLinuxProfile.cs" />
-    <Compile Include="Version2017_08_31\Models\ContainerServiceMasterProfile.cs" />
-    <Compile Include="Version2017_08_31\Models\ContainerServiceServicePrincipalProfile.cs" />
-    <Compile Include="Version2017_08_31\Models\ContainerServiceSshConfiguration.cs" />
-    <Compile Include="Version2017_08_31\Models\ContainerServiceSshPublicKey.cs" />
-    <Compile Include="Version2017_08_31\Models\ContainerServiceStorageProfileTypes.cs" />
-    <Compile Include="Version2017_08_31\Models\ContainerServiceVMDiagnostics.cs" />
-    <Compile Include="Version2017_08_31\Models\ContainerServiceVMSizeTypes.cs" />
-    <Compile Include="Version2017_08_31\Models\ContainerServiceWindowsProfile.cs" />
-    <Compile Include="Version2017_08_31\Models\KeyVaultSecretRef.cs" />
-    <Compile Include="Version2017_08_31\Models\ManagedCluster.cs" />
-    <Compile Include="Version2017_08_31\Models\ManagedClusterAccessProfile.cs" />
-    <Compile Include="Version2017_08_31\Models\ManagedClusterPoolUpgradeProfile.cs" />
-    <Compile Include="Version2017_08_31\Models\ManagedClusterUpgradeProfile.cs" />
-    <Compile Include="Version2017_08_31\Models\OrchestratorProfile.cs" />
-    <Compile Include="Version2017_08_31\Models\OSType.cs" />
-    <Compile Include="Version2017_08_31\Models\Page.cs" />
-    <Compile Include="Version2017_08_31\Models\Resource.cs" />
+    <Compile Remove="Properties\AssemblyInfo.cs" />
+    <EmbeddedResource Remove="Properties\AssemblyInfo.cs" />
+    <None Remove="Properties\AssemblyInfo.cs" />
+    <Content Remove="Properties\AssemblyInfo.cs" />
```

Comment:
Any csproj (should be all clients) that contain this block at the bottom, you can simply remove this ItemGroup and delete the AssemblyInfo.cs for this project. This was done because net452 used the AssemblyInfo.cs and netstandard20 projects did not. So, removing this block and deleting `AssemblyInfo.cs` should be done for all clients.